mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -860,23 +853,17 @@ public void testIcebergAndHmsTableProperties() throws Exception {
           shell.getHiveConf(), icebergTable, FileFormat.PARQUET, null, records);
       hmsParams = shell.metastore().getTable("default", "customers").getParameters();
       Map<String, String> summary = icebergTable.currentSnapshot().summary();
-      Assert.assertEquals(
-          summary.get(SnapshotSummary.TOTAL_DATA_FILES_PROP),
-          hmsParams.get(StatsSetupConst.NUM_FILES));
-      Assert.assertEquals(
-          summary.get(SnapshotSummary.TOTAL_RECORDS_PROP),
-          hmsParams.get(StatsSetupConst.ROW_COUNT));
-      Assert.assertEquals(
-          summary.get(SnapshotSummary.TOTAL_FILE_SIZE_PROP),
-          hmsParams.get(StatsSetupConst.TOTAL_SIZE));
+      assertThat(hmsParams).doesNotContainKey("custom_property");
```

Comment:
this seems wrong. Shouldn't this do the same comparison as the previous assertions were doing? 
```
assertThat(hmsParams)
    .containsEntry(StatsSetupConst.NUM_FILES, summary.get(SnapshotSummary.TOTAL_DATA_FILES_PROP))
    .containsEntry(StatsSetupConst.ROW_COUNT, summary.get(SnapshotSummary.TOTAL_RECORDS_PROP))
    .containsEntry(...)
```